mobile/src/components/uam-groups-create/helpers/group-creation-get-user-columns/group-creation-get-user-columns.helper.ts

```diff
@@ -0,0 +1,26 @@
+import { TableColumn } from '~/common/types/ui/table-column.type';
+import { UserTableAccessor } from '~/components/uam-groups-create/common/enums/enums';
+import { UsersTableRow } from '~/components/uam-groups-create/common/types/types';
+
+const getUserColumns = (): TableColumn<UsersTableRow>[] => {
```

Comment:
get-users-columns/get-users-columns.helper.ts
```suggestion
const getUsersColumns = (): TableColumn<UsersTableRow>[] => {
```